layouts/shortcodes/capture.html

```diff
@@ -3,5 +3,8 @@
 {{- if not $id -}}
 {{- errorf "missing id in capture" -}}
 {{- end -}}
-{{- $capture_id := printf "__cid_%s" $id -}}
-{{- .Page.Scratch.Set $capture_id .Inner -}}
\ No newline at end of file
+{{- $capture_id := printf "capture %s" $id -}}
+<!--
```

Comment:
@sftim, Things look good. Would you remove the comments? I'd like to see a clean build/deploy preview
before merging. Thanks!

Reply:
Removed!